shared/css/user-progress.scss

```diff
@@ -28,6 +28,11 @@
   color: $charcoal !important;
 }
 
+.level_link.locked, .level_link.locked:link, .level_link.locked:visited {
+  background-color: $level_not_tried;
```

Comment:
That said, do locked levels always show as charcoal text on white background, or can they change depending on attempted status?